OrbitCore/LibunwindstackUnwinder.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <asm/perf_regs.h>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "unwindstack/MachineX86_64.h"
+#include "unwindstack/RegsX86_64.h"
+#include "unwindstack/Unwinder.h"
+
+class LibunwindstackUnwinder {
+ public:
+  bool SetMaps(const std::string& maps_buffer);
+
+  [[nodiscard]] std::vector<unwindstack::FrameData> Unwind(
+      const std::array<uint64_t, PERF_REG_X86_64_MAX>& perf_regs,
+      const char* stack_dump, uint64_t stack_dump_size);
+
+ private:
+  std::unique_ptr<unwindstack::BufferMaps> maps_{nullptr};
+
+  static constexpr size_t MAX_FRAMES = 1024;
```

Comment:
Any reason why to pick 1024? Maybe add a comment if this choice is rather arbitrary.